### ISSUE TITLE
FormGroup: fastNavigation: true property on FormGroup is breaking the A11y rule when it's nested inside the <Card> component

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents#11452]
### Bug Description

We are encountering an accessibility issue when using the` <FormGroup>` component inside a `<Card>` component with fastNavigation enabled.

Current Implementation:

In our SFSF application, we commonly nest components in the following structure:

```
<Card>
  <FormGroup />
  <Bar />
<Card>
```

However, when fastNavigation is enabled, the expected group skipping behavior is not working as intended.

**Observed Behavior:**

When performing group skipping on a `<Card>`, the focus incorrectly:

Moves to the "Pencil" icon (as expected)[Please refer to attached screenshot],

But also moves into the first interactive element inside `<FormGroup>` (e.g., the [***** Show] element).[ Please refer to attached Screenshort]

This violates accessibility expectations because the focus should not enter nested groups during group navigation.

**Root Cause**
The root issue is that `<FormGroup>` has fastNavigation: true enabled by default, as seen here: https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/FormGroup.ts#L38

This default behavior overrides the intended group navigation flow when used inside a parent container like `<Card>`.

**Suggested Solution**
There should be a way to disable or override fastNavigation on a per-instance basis, especially when a component like `<FormGroup>`, `<List>`, or `<Bar>` is used inside containers such as `<Card>`.

Recommendation:

Expose a property (e.g., disableFastNavigation or fastNavigation="false") in components like `<FormGroup>`, `<List>`, and `<Bar>`.

Allow parent components to opt-out of default fastNavigation behavior when it's not appropriate in context.

### Expected Behaviour

**Expected Behavior:**

During group skipping:

Focus should only land on the "Pencil" icon in the card.

Then move directly to the next card, completely skipping the internal content of <FormGroup>.

### Log Output, Stack Trace or Screenshots

<img width="402" alt="Image" src="https://github.com/user-attachments/assets/df9fe276-3c9c-45e6-8ef5-c5da19576a42" />

### Priority

High

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by nnaydenow** [comment]
Hi @VidhiNPatel,

Fast navigation groups can indeed be nested. When a user triggers the fast navigation shortcut (e.g., F6) from within a nested group, the focus will continue along the tab chain until it reaches a control that belongs to a different group. If no adjacent nested group exists, the focus moves to the next control in the parent group that follows the nested group. As a result, it’s expected that controls other than the first one in a group may receive focus.

To help developers manage this behaviour, we can clarify in the documentation which components have fast navigation enabled by default. Also, we will enable mechanism to suppress this behaviour for a specific group or component by explicitly set the attribute `data-sap-ui-fastnavgroup="false"` on the element that need to be suppressed.

**Comment by VidhiNPatel** [comment]
Thanks for the clarification @nnaydenow on how fast navigation groups behave. Could you please share if there’s a timeline or plan for when you intend to implement the ability to suppress this behavior using `data-sap-ui-fastnavgroup="false"`?
